packages/react-components/icons/RadioButton.tsx

```diff
@@ -0,0 +1,50 @@
+import colors from '@celo/react-components/styles/colors'
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import Svg, { Circle } from 'svgs'
+
+
+interface Props {
+  height?: number
+  color?: string
+  selected?: boolean
+  disabled?:boolean
+}
+
+const styles = StyleSheet.create({
+  circle: {
+    paddingTop: 2,
+    height: 20,
+    width: 20,
+    borderRadius: 12,
+    borderWidth: 2,
+    borderColor: colors.gray4,
+  },
+})
+
+export default class RadioButton extends React.PureComponent<Props> {
+  static defaultProps = {
+    width: 20,
+    height: 20,
+    color: colors.greenUI,
+    selected: true, 
+    disabled: false,
```

Comment:
I don't see how disabled is being incorporated into the component yet. Shouldn't this impact the Svg somehow?